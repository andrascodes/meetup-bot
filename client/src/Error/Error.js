import React from 'react'

export const Error = props => (
  <div className="container-fluid">
    <h2>Error. Try again later!</h2>
  </div>
)