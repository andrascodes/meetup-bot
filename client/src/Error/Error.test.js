import React from 'react'
import renderer from 'react-test-renderer'

import { Error } from './Error'

describe('Error', () => {

  test('renders correctly with or without specifying props.text', () => {
    let tree = renderer.create(
      <Error 
      />
    ).toJSON()
    expect(tree).toMatchSnapshot()

    tree = renderer.create(
      <Error 
        text="Please try again later!"
      />
    )
    expect(tree).toMatchSnapshot()
  })

})