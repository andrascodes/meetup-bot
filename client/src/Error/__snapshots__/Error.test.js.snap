exports[`Error renders correctly with or without specifying props.text 1`] = `
<div
  className="container-fluid">
  <h2>
    Error. 
    Try again later!
  </h2>
</div>
`;

exports[`Error renders correctly with or without specifying props.text 2`] = `
<div
  className="container-fluid">
  <h2>
    Error. 
    Please try again later!
  </h2>
</div>
`;
